content/a/ppw-hardcore-friday-the-13th-tokio.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "PpW Hardcore Friday the 13th 東京"
+template = "article.html"
+weight = 0
+authors = ["M3n747"]
+[extra.gallery]
++++
+
+PpW Hardcore Friday the 13th 東京 (_Tokio_) is an upcoming show by [PpW Ewenement Wrestling](@/o/ppw.md), set to take place on 13.03.2026 in Tokio, in a venue called ["Tokyo Square in Itabashi"][are-wa-tokyo-square-desu]. The show will be a cooperation with [Big Japan Pro Wrestling][bjw] and other local promotions.
+
+<!-- more -->
+
+On 6.12.2025 Shigehiro Irie and Akane Fujita appeared at [Duch Świąt, Smak Krwi](@/e/ppw/2025-12-06-ppw-duch-swiat-smak-krwi.md). Although PpW made no official statement, it is a possibility that the deal came to be through the two guests who'd worked for multiple Japanese federations, including in the Tokyo Square.
+
+## Build-up
+
+* TBD
+
+{{ skip_card() }}
+
+### Recap
+
+* TBD
+
+## References
+
+* [Facebook post announcing the event][ppw-w-tokio-fb]
+* [Instagram post announcing the event][ppw-w-tokio-ig]
+
+[bjw]: https://en.wikipedia.org/wiki/Big_Japan_Pro_Wrestling
+[ppw-w-tokio-fb]: https://www.facebook.com/OficjalnePPW/posts/pfbid02cR3qT3LUn7PyuMBzjQThGFabsUGgrzSjiozvVCjEXzEzYC8y7MqzvMLyTZyaAioXl
+[ppw-w-tokio-ig]: https://www.instagram.com/ppw_hw/p/DSQO4rcCkes/
+[are-wa-tokyo-square-desu]: https://www.google.co.jp/maps/place/6-ch%C5%8Dme-8-10+Maenoch%C5%8D,+Itabashi+City,+Tokyo+174-0063,+Japonia/@35.7654889,139.6860555,20z/data=!4m16!1m9!3m8!1s0x6018ecd5d885e335:0xdc0f0c905124b45!2s6-ch%C5%8Dme-8-10+Maenoch%C5%8D,+Itabashi+City,+Tokyo+174-0063,+Japonia!3b1!8m2!3d35.7654955!4d139.6860525!10e5!16s%2Fg%2F12hk8xr5_!3m5!1s0x6018ecd5d885e335:0xdc0f0c905124b45!8m2!3d35.7654955!4d139.6860525!16s%2Fg%2F12hk8xr5_?entry=ttu&g_ep=EgoyMDI1MTIwOS4wIKXMDSoKLDEwMDc5MjA2OUgBUAM%3D
```

Comment:
My preference is OpenStreetMap (non-corporate web if possible) but it doesn't have a POI at that location, and is harder to use because everything is in Japanese.